qubesadmin/tests/mock_app.py

```diff
@@ -151,16 +158,21 @@ def default_string(self):
 
 DEFAULT_DOM0_PROPERTIES = {
     "clockvm": Property("", "vm", False),
+    "default_audiovm": Property("", "vm", False),
     "default_dispvm": Property("", "vm", False),
     "default_kernel": Property("1.1", "str", True),
     "default_netvm": Property("", "vm", False),
     "default_pool": Property("file", "str", True),
+    "default_pool_private": Property("vm-pool", "str", True),
```

Comment:
Global properties are not the same as dom0 properties. You handle them separately later, so it's logical to keep them in separate dicts here too.

Reply:
yeah, you are right, this ended up really messy because I got confused with default_dispvm being a global property and a vm property at the same time. But hopefully I fixed it (will push in a sec)